cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -263,9 +273,42 @@ protected CameraPreview instantiatePreview(Context context, ViewGroup container)
     protected void onAttachedToWindow() {
         super.onAttachedToWindow();
         if (mCameraPreview == null) {
+
+            // check if we have any overlay view before instantiating preview
```

Comment:
As said in another comment (maybe below), let's assume this is always true. And CameraPreviews should not know about the presence of overlays. 

This was needed before because you were creating the new texture in the preview, but we won't do this anymore